### PR TITLE
Trying to equip an enemy radio will make an explosion

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Boom.

## Why It's Good For The Game

Cannot steal the enemy comms

## Changelog
:cl:
expansion: You can no longer still enemy radios, they explode when trying to equip
/:cl:

### CODE REVIEW

**Comment by EvilMelonTheSecond** [comment]
Won't they still work when held in hand or storage? I guess it would be fun to listen to comms like a radio in hand though.

**Comment by BraveMole** [comment]
> Won't they still work when held in hand or storage? I guess it would be fun to listen to comms like a radio in hand though.

Needs to be in your ear :)

**Comment by KornFlaks** [comment]
Command gives us radios that can be remotely detonated.

Is command trying to kill us all?

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Make a tochat

File: code/game/objects/items/radio/headset.dm [hunk 1]
```diff
@@ -178,6 +178,11 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 
 /obj/item/radio/headset/mainship/equipped(mob/living/carbon/human/user, slot)
 	if(slot == SLOT_EARS)
+		if(GLOB.faction_to_freq[user.faction] != frequency)
+			playsound(user, 'sound/effects/bang.ogg', 50, 1)
```
Comment (BraveMole) [inline code/game/objects/items/radio/headset.dm:182]:
```suggestion
			to_chat(user, span_warning("[src] violently buzzes and explodes in your ear"))
			playsound(user, 'sound/effects/bang.ogg', 50, 1)
```

File: code/game/objects/items/radio/headset.dm [hunk 2]
```diff
@@ -178,6 +178,12 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 
 /obj/item/radio/headset/mainship/equipped(mob/living/carbon/human/user, slot)
 	if(slot == SLOT_EARS)
+		if(GLOB.faction_to_freq[user.faction] != frequency)
+			to_chat(user, span_warning("[src] violently buzzes and explodes in your ear"))
```
Comment (TiviPlus) [inline code/game/objects/items/radio/headset.dm:182]:
```suggestion
			to_chat(user, span_warning("[src] violently buzzes and explodes in your face as it's tampering mechanisms are triggered!"))
```

File: code/game/objects/items/radio/headset.dm [hunk 3]
```diff
@@ -178,6 +178,12 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 
 /obj/item/radio/headset/mainship/equipped(mob/living/carbon/human/user, slot)
 	if(slot == SLOT_EARS)
+		if(GLOB.faction_to_freq[user.faction] != frequency)
+			to_chat(user, span_warning("[src] violently buzzes and explodes in your face as it's tampering mechanisms are triggered!"))
```
Comment (BraveMole) [inline code/game/objects/items/radio/headset.dm:182]:
```suggestion
			to_chat(user, span_warning("[src] violently buzzes and explodes in your face as its tampering mechanisms are triggered!"))
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**